tex/gsp-default.tex

```diff
@@ -47,7 +49,7 @@
 
 %% These macro enable the tuning of linepenalty, tolerance, pretolerance
 %% and emergencystretch
-\def\grelooseness{\looseness}%
+\def\grelooseness{-1}%
```

Comment:
It seems incongruous given the way we've been doing things to have the user redefine a macro to change the behavior, so at some point, we should probably change this to an internal macro that can be changed via a "setter" macro.  This change is fine for a bugfix release.